app/src/main/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
+
+import { app, BrowserWindow } from 'electron'
+import * as path from 'path'
+import { format as formatUrl } from 'url'
+
+const is_development = process.env.NODE_ENV !== 'production'
+
+let main_window_keep_alive
+
+function create_main_window() {
+    const window = new BrowserWindow(
+        {
+            width          : 1024,
+            height         : 768,
+            frame          : false,
+        }
+    )
+
+    var search = "";
```

Comment:
I would use better name here. As far as I see this is neither a search query not a search result. It is a scene name right?